sw/host/cryptotest/testvectors/data/BUILD

```diff
@@ -128,3 +128,100 @@ genrule(
         ("ecdh_secp384r1_test.json", "wycheproof_ecdh_p384"),
     ]
 ]
+
+[
+    run_binary(
+        name = "nist_cavp_aes_{}_{}_json".format(
+            function_name.lower(),
+            keylen_with_tmode,
+        ),
+        srcs = [
+            "@nist_cavp_aes_kw_sp_800_38f//:kwtestvectors/{}_{}.txt".format(
+                function_name,
+                keylen_with_tmode,
+            ),
+            "//sw/host/cryptotest/testvectors/data/schemas:aes_kw_schema.json",
+        ],
+        outs = [":nist_cavp_aes_{}_{}.json".format(
+            function_name.lower(),
+            keylen_with_tmode,
+        )],
+        args = [
+            "--src",
+            "$(location @nist_cavp_aes_kw_sp_800_38f//:kwtestvectors/{}_{}.txt)".format(function_name, keylen_with_tmode),
```

Comment:
If all the test vectors you need are in this directory, it's cleaner to add `strip_prefix = "kwtestvectors"` in the `http_archive` pull for the test vector archive, and then you can reference all the test vector files like they are in the root directory of the bazel collection.